fluent/migrate/transforms.py

```diff
@@ -236,6 +236,13 @@ def __call__(self, ctx):
         selector = evaluate(ctx, self.selector)
         variants = value.split(';')
         keys = ctx.plural_categories
+
+        # A special case for languages with one plural category. We don't need
+        # to insert a SelectExpression at all for them.
+        if len(keys) == len(variants) == 1:
```

Comment:
The special case is only triggered when _both_ the number of plural categories in CLDR and the number of variants in the legacy translation are `1`.

If the legacy translation has more variants we will still insert a `SelectExpression`: https://github.com/projectfluent/python-fluent/pull/27/files#diff-6d4522d85d5c0eb1cbe1e024d2407986R69